SU-HSDO/suhumsci
# Release: FourKitchens Sprint 10

* Mega menu V1 keyboard improvements - https://fourkitchens.atlassian.net/browse/SHS-22

* Update class for breakout box color - https://fourkitchens.atlassian.net/browse/SHS-55

# READY FOR REVIEW

## Summary
- Improves keyboard navigation and focus styles for the mega menu

## Need Review By (Date)
9/28

## Urgency
low

## Steps to Test
1. Run `lando blt drupal:sync --site=music` or setup the music site on your local as this site in particular uses the mega menu
2. Using the keyboard, tab through the main navigation and ensure that any menu item that is a link and a button (Training for me on Traditional) that the button now receives an outline
3. Edit any page and add `Text Area` component. Verify there is a "Breakout Box" option under the "Styles Dropdown"


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
